doc/releases/release-notes-3.5.rst

```diff
@@ -45,6 +45,18 @@ Architectures
 
 * RISC-V
 
+  * Added support for detecting null pointer exception using PMP.
+  * Renamed :kconfig:option:`CONFIG_RISCV_MTVEC_VECTORED_MODE` to
+    :kconfig:option:`CONFIG_RISCV_VECTORED_MODE`.
```

Comment:
this might be better suited for the migration guide? /cc @carlescufi

Reply:
Hi! Judging from the migration guide's content, it indeed looks like the highlighted point should be added there. Should I remove it from release notes as well?

Reply:
> Hi! Judging from the migration guide's content, it indeed looks like the highlighted point should be added there. Should I remove it from release notes as well?

Yes, as per guidance from @carlescufi we shouldn't duplicate information in the two files

Reply:
Ok. I've removed it from the release notes and now this point is only present in the migration guide.